openwisp_users/api/permissions.py

```diff
@@ -66,3 +68,32 @@ class IsOrganizationOwner(BaseOrganizationPermission):
 
     def validate_membership(self, user, org):
         return org and (user.is_superuser or user.is_owner(org))
+
+
+class CustomDjangoModelPermissions(DjangoModelPermissions):
```

Comment:
Just wondering, can we name it something like `ViewDjangoModelPermissions` so that the name makes more sense in the other modules about that's "Custom" about this class!

Reply:
Done @atb00ker :+1: